reset.sh

```diff
@@ -59,4 +59,4 @@ if [ -n "$version" ]; then
 fi
 
 # Install.
-exec ./install.sh
+./install.sh
```

Comment:
What was the original intent of using `exec`?

This piece of logic might be usable instead of `source "$(dirname $0)/_min-requirements.sh"`:
https://github.com/getsentry/sentry/blob/6776229fbde4ca8cd52884d369a1b498a43dc6b4/scripts/do.sh#L5-L10

Reply:
I didn't have a strong reason to use `exec` originally. I did it that way because conceptually we are done with the one script so figured we'd hand off to the next, didn't think through the nuances.

I've started trying to whittle down a better repro in order to understand better what's going on and what the best solution is. That's turning out to take more time than I'd like, though. This `reset.sh` script is hacky and mostly for me. The real bug I want to focus on is #1133.

You okay to merge as-is?